### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
Created the local storage with javascript

### CODE REVIEW

File: java.js [hunk 1]
```diff
@@ -183,4 +185,25 @@ function checkLowerCase(input) {
...
+  email.value = e;
+  text.value = t;
+}
+
+valueInput.forEach(showNow);
+
+form.addEventListener('change', (event) => {
+  event.preventDefault();
+  addVale(name.value, email.value, text.value);
+});
```
Comment (GzimAsani) [inline java.js]: - [ ] Your form is not being persistent to refresh, kindly check the video below :arrow_double_down: 
image

https://www.loom.com/share/afb3a3e3670f4ee2a28eb56c61be8f8e

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @Uchenna-ibute and @danielmatama                                 :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your styling looks great +1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: java.js [hunk 2]
```diff
@@ -183,4 +185,25 @@ function checkLowerCase(input) {
...
+  return { n, e, t };
+}
+
+function showNow({ n, e, t }) {
+  namee.value = n;
+  email.value = e;
+  text.value = t;
+}
+
+valueInput.forEach(showNow);
```
Comment (amejid) [inline java.js:204]: - [ ] Kindly make sure the form data is persisted between refreshes as requested by the previous reviewer. 🤝️
This error is shown in the console, please take a look at it.

Screenshot from 2022-12-23 20-59-13

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linter checks passing :heavy_check_mark: 
- Nice UI design :fire: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: java.js (hunk 2 above)
Comment (Uchenna-ibute) [reply java.js:204]: Hello
Thank you for the feedback, but I don't understand. I have run this code in my browser, its not showing any [errors](pp)

**Comment by Uchenna-ibute** [comment]
Thank you for the feedback.
But I don't understand what the problem is, I have run the code in my
browser but I can not find the error you mentioned.

**Comment by amejid** [comment]
@Uchenna-ibute Please make sure you are in the correct branch and run the project. Can you share a demonstration (video) of how it behaves on your machine so we can understand the issue?

**Comment by Uchenna-ibute** [comment]
https://www.loom.com/share/a16ab837ffbf4f448d24998d2739bb85
This is the video's link.

**Pull request merged**